includes/wikia/tasks/AsyncTaskList.class.php

```diff
@@ -326,6 +325,8 @@ public function queue( AMQPChannel $channel = null ) {
 
 	/**
 	 * @return AMQPConnection connection to message broker
+	 * @throws AMQPRuntimeException
+	 * @throws AMQPTimeoutException
 	 */
 	protected function connection() {
```

Comment:
Btw. this is not introduced within this PR but perhaps name of this method should be different - getConnection()